docs/static/js/validator.js

```diff
@@ -462,7 +462,14 @@ class Validator
 
         let errors = [];
         if ( !this._validate_internal(data) )
-            errors = this._validate_internal.errors.map(e => this._cleaned_error(e));
+            errors = this._validate_internal.errors
+                .map(e => this._cleaned_error(e))
+                .map(e => {
+                    return {
+                        ...e,
+                        path_names: this._get_friendly_path_names(e.path, data)
```

Comment:
Might be worth adding to _cleaned_error to avoid a second pass with `map()`

Reply:
Done